tools/rules/heron_tools.bzl

```diff
@@ -4,6 +4,15 @@
 def heron_tools_files():
     return heron_tools_bin_files() + heron_tools_conf_files() + heron_tools_lib_files()
 
+def heron_tools_bin_all_files():
+    return [
+        "//heron/tools/tracker/src/python:heron-tracker",
+        "//heron/tools/ui/src/python:heron-ui",
+        "//heron/tools/apiserver/src/shell:heron-apiserver",
+        "//heron/tools/cli/src/python:heron",
+        "//heron/tools/explorer/src/python:heron-explorer",
+    ]
+
 def heron_tools_bin_files():
```

Comment:
is this needed anymore?

Reply:
@cckellogg - you mean heron_tools_bin_files - probably leave it for now until we don't need it.

Reply:
Yes the heron_tools_bin_files it looks like the new one replaces it.

Reply:
yes for now - Let us leave the old one and remove it later.

Reply:
@cckellogg - we need both since one is used for heron binary packaging while the other is used for docker container.